src/pages/ParkingLane/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import styled from 'styled-components';
+import { ArticleHeader, ArticleWrapper, Intro } from '~/components2/Article';
+import LogoFile from './assets/wappen.png';
+import LogoFile2 from './assets/wappen@2x.png';
+import LogoFile3 from './assets/wappen@3x.png';
```

Comment:
I think it would be best, if we use a SVG variant, like https://commons.wikimedia.org/wiki/File:Coat_of_arms_of_borough_Friedrichshain-Kreuzberg.svg